docs/src/test-runners-csharp.md

```diff
@@ -58,6 +60,10 @@ public class MyTest : PageTest
 }
 ```
 
+:::note
```

Comment:
can we move it under the "### Running NUnit tests in Parallel" heading and don't have a Note for it?

Rest lgtm.

Reply:
How about now @mxschmitt ?

Reply:
amazing!